src/DataShare/DataShare/help/Get-AzDataShareAccount.md

```diff
@@ -0,0 +1,178 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataShare.dll-Help.xml
+online version: https://docs.microsoft.com/en-us/powershell/module/az.datashare/get-azdatashareaccount
+schema: 2.0.0
+---
+
+# Get-AzDataShareAccount
+
+## SYNOPSIS
+Gets information about DataShare Accounts
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
```

Comment:
@gaurangisaxena please regenerate the help files for your cmdlets based on the latest changes made to your cmdlets

Reply:
updated help files